[2021-11-01 05:25:36,937] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: docker_dag.t1 scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2021-11-01 05:25:37,082] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: docker_dag.t1 scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2021-11-01 05:25:37,082] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 05:25:37,082] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-01 05:25:37,082] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 05:25:37,270] {taskinstance.py:1262} INFO - Executing <Task(_PythonDecoratedOperator): t1> on 2021-10-31 00:00:00+00:00
[2021-11-01 05:25:37,291] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'docker_dag', 't1', 'scheduled__2021-10-31T00:00:00+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/docker_dag.py', '--cfg-path', '/tmp/tmpfxs9k7ad', '--error-file', '/tmp/tmp_29_h4ds']
[2021-11-01 05:25:37,422] {standard_task_runner.py:77} INFO - Job 53: Subtask t1
[2021-11-01 05:25:37,367] {standard_task_runner.py:52} INFO - Started process 281 to run task
[2021-11-01 05:25:37,956] {logging_mixin.py:109} INFO - Running <TaskInstance: docker_dag.t1 scheduled__2021-10-31T00:00:00+00:00 [running]> on host 8f906900190b
[2021-11-01 05:25:38,312] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=docker_dag
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2021-10-31T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-31T00:00:00+00:00
[2021-11-01 05:25:38,313] {python.py:152} INFO - Done. Returned value was: None
[2021-11-01 05:25:38,389] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=docker_dag, task_id=t1, execution_date=20211031T000000, start_date=20211101T052537, end_date=20211101T052538
[2021-11-01 05:25:38,515] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-01 05:25:38,904] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-11-01 05:33:44,785] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: docker_dag.t1 manual__2021-11-01T05:33:39.188875+00:00 [queued]>
[2021-11-01 05:33:45,127] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: docker_dag.t1 manual__2021-11-01T05:33:39.188875+00:00 [queued]>
[2021-11-01 05:33:45,127] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 05:33:45,127] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-01 05:33:45,127] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 05:33:45,146] {taskinstance.py:1262} INFO - Executing <Task(_PythonDecoratedOperator): t1> on 2021-11-01 05:33:39.188875+00:00
[2021-11-01 05:33:45,151] {standard_task_runner.py:52} INFO - Started process 185 to run task
[2021-11-01 05:33:45,216] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'docker_dag', 't1', 'manual__2021-11-01T05:33:39.188875+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/docker_dag.py', '--cfg-path', '/tmp/tmpzc136020', '--error-file', '/tmp/tmplfc8zthi']
[2021-11-01 05:33:45,245] {standard_task_runner.py:77} INFO - Job 57: Subtask t1
[2021-11-01 05:33:45,438] {logging_mixin.py:109} INFO - Running <TaskInstance: docker_dag.t1 manual__2021-11-01T05:33:39.188875+00:00 [running]> on host a26aba8cea96
[2021-11-01 05:33:45,721] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=docker_dag
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2021-11-01T05:33:39.188875+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-01T05:33:39.188875+00:00
[2021-11-01 05:33:45,721] {python.py:152} INFO - Done. Returned value was: None
[2021-11-01 05:33:45,751] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=docker_dag, task_id=t1, execution_date=20211101T053339, start_date=20211101T053344, end_date=20211101T053345
[2021-11-01 05:33:45,877] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-01 05:33:45,993] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-11-01 05:39:47,467] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: docker_dag.t1 manual__2021-11-01T05:39:42.209376+00:00 [queued]>
[2021-11-01 05:39:47,581] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: docker_dag.t1 manual__2021-11-01T05:39:42.209376+00:00 [queued]>
[2021-11-01 05:39:47,627] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 05:39:47,628] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-01 05:39:47,628] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 05:39:47,674] {taskinstance.py:1262} INFO - Executing <Task(_PythonDecoratedOperator): t1> on 2021-11-01 05:39:42.209376+00:00
[2021-11-01 05:39:47,719] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'docker_dag', 't1', 'manual__2021-11-01T05:39:42.209376+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/docker_dag.py', '--cfg-path', '/tmp/tmpefqdr1p5', '--error-file', '/tmp/tmpcxf4s1ym']
[2021-11-01 05:39:47,755] {standard_task_runner.py:77} INFO - Job 59: Subtask t1
[2021-11-01 05:39:47,678] {standard_task_runner.py:52} INFO - Started process 390 to run task
[2021-11-01 05:39:48,075] {logging_mixin.py:109} INFO - Running <TaskInstance: docker_dag.t1 manual__2021-11-01T05:39:42.209376+00:00 [running]> on host a26aba8cea96
[2021-11-01 05:39:48,473] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=docker_dag
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2021-11-01T05:39:42.209376+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-01T05:39:42.209376+00:00
[2021-11-01 05:39:48,474] {python.py:152} INFO - Done. Returned value was: None
[2021-11-01 05:39:48,809] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=docker_dag, task_id=t1, execution_date=20211101T053942, start_date=20211101T053947, end_date=20211101T053948
[2021-11-01 05:39:49,119] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-01 05:39:49,406] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-11-01 05:40:32,360] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: docker_dag.t1 manual__2021-11-01T05:40:30.168754+00:00 [queued]>
[2021-11-01 05:40:32,421] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: docker_dag.t1 manual__2021-11-01T05:40:30.168754+00:00 [queued]>
[2021-11-01 05:40:32,421] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 05:40:32,421] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-01 05:40:32,421] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 05:40:32,473] {taskinstance.py:1262} INFO - Executing <Task(_PythonDecoratedOperator): t1> on 2021-11-01 05:40:30.168754+00:00
[2021-11-01 05:40:32,512] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'docker_dag', 't1', 'manual__2021-11-01T05:40:30.168754+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/docker_dag.py', '--cfg-path', '/tmp/tmpu0tx89ke', '--error-file', '/tmp/tmpcc8oo8ny']
[2021-11-01 05:40:32,581] {standard_task_runner.py:77} INFO - Job 61: Subtask t1
[2021-11-01 05:40:32,529] {standard_task_runner.py:52} INFO - Started process 425 to run task
[2021-11-01 05:40:33,380] {logging_mixin.py:109} INFO - Running <TaskInstance: docker_dag.t1 manual__2021-11-01T05:40:30.168754+00:00 [running]> on host a26aba8cea96
[2021-11-01 05:40:33,703] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=docker_dag
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2021-11-01T05:40:30.168754+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-01T05:40:30.168754+00:00
[2021-11-01 05:40:33,710] {python.py:152} INFO - Done. Returned value was: None
[2021-11-01 05:40:33,762] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=docker_dag, task_id=t1, execution_date=20211101T054030, start_date=20211101T054032, end_date=20211101T054033
[2021-11-01 05:40:33,850] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-01 05:40:34,064] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-11-01 06:07:52,786] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: docker_dag.t1 manual__2021-11-01T06:06:53.703284+00:00 [queued]>
[2021-11-01 06:07:54,437] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: docker_dag.t1 manual__2021-11-01T06:06:53.703284+00:00 [queued]>
[2021-11-01 06:07:54,438] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 06:07:54,438] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-01 06:07:54,438] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 06:07:55,284] {taskinstance.py:1262} INFO - Executing <Task(_PythonDecoratedOperator): t1> on 2021-11-01 06:06:53.703284+00:00
[2021-11-01 06:07:55,614] {standard_task_runner.py:52} INFO - Started process 1310 to run task
[2021-11-01 06:07:55,393] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'docker_dag', 't1', 'manual__2021-11-01T06:06:53.703284+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/docker_dag.py', '--cfg-path', '/tmp/tmpqsk0vlmj', '--error-file', '/tmp/tmp_ajfolpf']
[2021-11-01 06:07:56,031] {standard_task_runner.py:77} INFO - Job 63: Subtask t1
[2021-11-01 06:08:01,289] {logging_mixin.py:109} INFO - Running <TaskInstance: docker_dag.t1 manual__2021-11-01T06:06:53.703284+00:00 [running]> on host a26aba8cea96
